packages/datepickers/src/elements/DatePicker/components/Calendar.tsx

```diff
@@ -94,13 +94,13 @@ export const Calendar = forwardRef<HTMLDivElement, ICalendarProps>(
       }
 
       return (
-        <StyledCalendarItem key={`day-${itemsIndex}`} isCompact={isCompact}>
+        <StyledCalendarItem key={`day-${itemsIndex}`} $isCompact={isCompact}>
           <StyledDay
-            isToday={isCurrentDate}
-            isPreviousMonth={isPreviousMonth}
-            isSelected={isSelected}
-            isDisabled={isDisabled}
-            isCompact={isCompact!}
+            $isToday={isCurrentDate}
+            $isPreviousMonth={isPreviousMonth}
+            $isSelected={isSelected}
+            $isDisabled={isDisabled}
```

Comment:
```suggestion
            [aria-selected]={isSelected || undefined}
            [aria-disabled]={isDisabled || undefined}
```

See https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/. Even though we won't be a11y compliant here, this will give us better CSS query support.

Reply:
Since the calendar isn't keyboard accessible in any form, I'm conflicted on if a11y updates like this give us any real value. Thoughts?